lib/mayaUsd/ufe/UsdRotatePivotTranslateUndoableCommand.cpp

```diff
@@ -20,22 +20,26 @@
 MAYAUSD_NS_DEF {
 namespace ufe {
 
-UsdRotatePivotTranslateUndoableCommand::UsdRotatePivotTranslateUndoableCommand(const UsdPrim& prim, const Ufe::Path& ufePath, const Ufe::SceneItem::Ptr& item)
-	: Ufe::TranslateUndoableCommand(item)
-	, fPrim(prim)
-	, fPath(ufePath)
+UsdRotatePivotTranslateUndoableCommand::UsdRotatePivotTranslateUndoableCommand(const Ufe::Path& path)
+	: Ufe::TranslateUndoableCommand(path)
+	, fPath(path)
 	, fNoPivotOp(false)
 {
+	// create on first access
+	if(!fItem) {
+	    fItem = std::dynamic_pointer_cast<UsdSceneItem>(Ufe::Hierarchy::createItem(fPath));
+	}
+
 	// Prim does not have a pivot translate attribute
 	const TfToken xpivot("xformOp:translate:pivot");
-	if (!fPrim.HasAttribute(xpivot))
+	if (!fItem->prim().HasAttribute(xpivot))
```

Comment:
Create inline accessor for prim:
inline UsdPrim prim() {
    return sceneItem->prim();
}